src/devices/controller/leds.cc

```diff
@@ -0,0 +1,11 @@
+#include "leds.h"
+
+std::array<CRGB, kLedCount> leds;
+
+void SetMainLed(uint8_t led_index, CRGB rgb) {
```

Comment:
What does "Main" mean here?

Reply:
Added a comment - it's for the 3 x 12 grid of LEDs.